transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/http/RestClientInvocation.java

```diff
@@ -116,12 +118,17 @@ public void invoke(Invocation invocation, AsyncResponse asyncResp) throws Except
         filter.beforeSendRequest(invocation, requestEx);
       }
     }
+    List<HttpRestClientInvocationHook> httpRestClientInvocationHooks =
+            SPIServiceUtils.getAllService(HttpRestClientInvocationHook.class);
 
     clientRequest.exceptionHandler(e -> {
       invocation.getTraceIdLogger()
           .error(LOGGER, "Failed to send request, alreadyFailed:{}, local:{}, remote:{}, message={}.",
               alreadyFailed, getLocalAddress(), ipPort.getSocketAddress(),
               ExceptionUtils.getExceptionMessageWithoutTrace(e));
+      httpRestClientInvocationHooks.forEach(httpServerExceptionHandler -> {
```

Comment:
Maybe you can add a custom handler to fillful your requirements. 

e.g. 'MyHandler implements Handler'

servicecomb.handler.chain.Consumer.defaut:  your_current_handlers,myhandler

myhandler can process the exception thrown in RestClientInvocation

Reply:
https://docs.servicecomb.io/java-chassis/zh_CN/references-handlers/intruduction/

I found it didn't meet my requirements through experiments.

 My handler executes before the exception is returned, and does not return to handler after the exception is thrown.

I can't get this exception message.

Is there any other way

Reply:
```
public class MyHandler implements Handler {

  private static final Logger LOGGER = LoggerFactory.getLogger(MyHandler.class);

  public static final String SPLITPARAM_RESPONSE_USER_SUFFIX = "(modified by MyHandler)";

  @Override
  public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
    LOGGER.info("before");

    invocation.next(response -> {
         LOGGER.info("after");
    });
  }
}
```

You can find many Handler implementations in servicecomb-java-chassis project.